pages/twitter.tsx

```diff
@@ -116,11 +117,19 @@ const Twitter: NextPage = () => {
         transactionData?.status &&
         !transactionError &&
         !transactionData?.status.includes("ACCEPTED") &&
-        transactionData?.status !== "PENDING"
+        transactionData?.status !== "REJECTED" &&
+        transactionData?.status !== "REVERTED"
       ) {
         posthog?.capture("twitterVerificationTx");
         addTransaction({
-          hash: twitterVerificationData?.transaction_hash ?? "",
+          timestamp: Date.now(),
+          subtext: "Twitter verification",
```

Comment:
So here you can say `Twitter verification on Starknet ID #X` ? Same for the other socials ? ID is the tokenId of the starknet ID